packages/neuron-ui/src/components/Transaction/index.tsx

```diff
@@ -178,14 +156,41 @@ const Transaction = () => {
         }
         const capacity = shannonToCKBFormatter(cell.capacity || '0')
 
+        // REFACTOR: should be refacter into a component
```

Comment:
I think it can be move to outside of the function component first.

Reply:
Indeed, I'll update it within this PR.

Reply:
> I think it can be move to outside of the function component first.

Indeed, I'll update it within this PR.

Reply:
Updated